packages/astro/src/core/build/plugins/plugin-manifest.ts

```diff
@@ -264,6 +243,14 @@ async function buildManifest(
 			styles,
 			routeData: serializeRouteData(route, settings.config.trailingSlash),
 		});
+
+		// Add the built .html file as a staticFile
+		if(route.prerender && route.pathname) {
+			const outFolder = getOutFolder(opts.settings, route.pathname, route);
+			const outFile = getOutFile(opts.settings.config, outFolder, route.pathname, route);
+			const file = outFile.toString().replace(opts.settings.config.build.client.toString(), '');
+			staticFiles.push(file);
+		}
```

Comment:
Is this correct? The `.toString()` method of `URL` returns the file **without trailing slash**, so if you replace it, you will end up with `file` being `/dist/client/file.html`, which is an absolute path. Is this what you expect/want? Shouldn't it be safer to use a relative path?